Content.Server/VoiceMask/VoiceMaskComponent.cs

```diff
@@ -6,4 +8,14 @@ public sealed partial class VoiceMaskComponent : Component
     [ViewVariables(VVAccess.ReadWrite)] public bool Enabled = true;
 
     [ViewVariables(VVAccess.ReadWrite)] public string VoiceName = "Unknown";
+
+    /// <summary>
+    /// If EnableSpeechVerbModification is true, overrides the speech verb used when this entity speaks.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [ValidatePrototypeId<SpeechVerbPrototype>]
+    public string SpeechVerb = "Default";
```

Comment:
Use ProtoId

Reply:
If I change it to a ProtoId, is it okay not to have the ValidatePrototypeId on it? I'd like to have it nullable and only use it if it's non-null, so I can get rid of EnableSpeechVerbModification.